src/rp2_common/pico_unique_id/include/pico/unique_id.h

```diff
@@ -41,6 +41,28 @@ extern "C" {
 
 #define PICO_UNIQUE_BOARD_ID_SIZE_BYTES 8
 
+/**
+ * \brief Static initialization order
+ * \ingroup pico_unique_id
+ *
+ * This defines the init_priority of the pico_unique_id. By default, it is 1000. The valid range is
+ * from 101-65535. Set to -1 to set the priority to none, thus putting it after 65535. Changing
```

Comment:
I'm afraid that static initializers aren't something I know much about, but would there ever be a reason that you'd _want_ to set this to -1 ?

Reply:
The current, existing behavior can be reverted to by setting to -1. I don't think there is a reason you couldn't use 65535 instead, but the default linker script order is that what I've called -1 is after 65535 (I could call it 65536, but I think that may be more confusing, oddly enough)

Linker script: https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_crt0/rp2040/memmap_default.ld#L79-L80

Note the `.ctor.*` are the ones with init_priority, and `.ctor` is the -1/default/old behavior

Reply:
If you would prefer, I can remove -1 and we then wait until someone hollers that we broke something? (I think that is unlikely, but not sure, as I haven't explicitly used init_priority before this year)